src/components/JsonSchemaViewer.tsx

```diff
@@ -106,6 +110,7 @@ const JsonSchemaViewerInner = ({
     const jsonSchemaTree = new JsonSchemaTree(schema, {
       mergeAllOf: true,
       refResolver: resolveRef,
+      maxRefDepth,
```

Comment:
Is it correct that `JsonSchemaTree` already had an optional `maxRefDepth` parameter, and this PR just allows the caller to specify that when instantiating a `JsonSchemaViewer` component?  (And if it's not specified, there's no limit?)

Reply:
it did not have the parameter, i added it and didn't set a limit for backwards compatability